index.html

```diff
@@ -1,5 +1,60 @@
 <html>
     <head>
+        <style>
+            [data-theme="light"] {
+                --color-background: white;
+                --color-text: black;
+                --color-marked: lightgray;
+                --color-next: cornflowerblue;
+                --color-bingo: lightgreen;
+                --color-border: grey;
+            }
+
+            [data-theme="dark"] {
+                --color-background: #222;
+                --color-text: white;
+                --color-marked: #555;
+                --color-next: #03346E;
+                --color-bingo: green;
+                --color-border: lightgrey;
+            }
+
+            body {
+                background-color: var(--color-background);
+                color: var(--color-text);
+            }
+
+            .bingo-grid {
+                display: inline-grid; 
+                grid-template-columns: repeat(5, 80px); 
+                grid-template-rows: repeat(5, 80px); 
+                border-top: 0.5px solid var(--color-border); 
+                border-left: 0.5px solid var(--color-border);
+            }
+
+            .grid-cell {
+                background-color: var(--color-background);
+                padding: 12px 0 0 27px; 
+                border-right: 0.5px solid var(--color-border); 
+                border-bottom: 0.5px solid var(--color-border);
+            }
+
+            .grid-cell[data-state="marked"] {
+                background-color: var(--color-marked);
+            }
+
+            .grid-cell[data-state="next"] {
+                background-color: var(--color-next);
+            }
+
+            .grid-cell[data-state="bingo"] {
+                background-color: var(--color-bingo);
+            }
+
+            .heatmap {

```

Comment:
I think it would be prettier if we used .grid-cell[data-state="heatmap"] and add a HEATMAP entry to checkboxStates.

Then we can remove the setCheckboxHeatmap function and just use setCheckboxState(HEATMAP/NONE) instead.

Reply:
Thanks for the idea, made this change.